### DISCUSSION TITLE
Madara community call #15

### DISCUSSION THREAD

**Discussion by tdelabro** [keep-starknet-strange/madara#1304] [Announcements]
## Agenda
- Hotstuff consensus algorithm

## Invitations
<a href="https://starkware-co.zoom.us/j/89039661605"><img src="https://img.shields.io/badge/Join%20on-Zoom-blue?logo=zoom" height="75" /></a>
<a href="https://calendar.google.com/calendar/u/0/r/eventedit/dGxndm04YmhlM2xxM3VwMGtmZnFkZjVqcjAgdGltb3RoZWVAc3Rhcmt3YXJlLmNv"><img src="https://ssl.gstatic.com/calendar/images/dynamiclogo_2020q4/calendar_31_2x.png" height="75" /></a>

## Date & Time
- When: [Friday, 8 December 2023 at 4 PM CEST](https://www.timeanddate.com/worldclock/converter.html?iso=20231208T150000&p1=195&p2=179&p3=1440&p4=176&p5=240)

> "Wherever There Is Light — There Are Always Shadows."

**Comment by Debashis24k** [comment]
Let's go

**Reply by Tigersame** [reply]
Go
